src/commands/freeze.js

```diff
@@ -0,0 +1,16 @@
+import freeze from '../scripts/freeze'
+import runWithTruffle from '../utils/runWithTruffle'
+
+module.exports = function(program) {
+  program
+    .command('freeze')
+    .usage('--network <network> [options]')
+    .description('Freeze current release version of your stdlib project.')
```

Comment:
Freeze can be invoked in app projects as well. Also, let @maraoz know after merging, since he's reviewing all CLI commands help now.

Reply:
It's fine with me if we want to ship it as command for libs only, but let's be sure about it. It would also affect whether this logic is in the Lib or BaseController.

Reply:
Scratch that. I thought that `AppDirectory` extended from `FreezableImplementationDirectory`, but it is not the case. Please disregard these comments!